res/notes/testplan.txt

```diff
@@ -1,19 +1,57 @@
-
 # Release Qualification Test Cases
 
-> Status: Work In Progress
+> Status: Work in Progress
```

Comment:
Please refrain from inconsequential edits.

I can't easily determine from the diff here what you've actually changed. By re-ordering lines, it's nearly impossible to determine what you've added/removed, and whether the changes are valuable to accept or if they introduce problems.